lib/taskana-core/src/main/java/pro/taskana/task/internal/jobs/TaskRefreshJob.java

```diff
@@ -1,15 +1,17 @@
-package pro.taskana.common.internal.jobs;
+package pro.taskana.task.internal.jobs;
 
 import java.util.Arrays;
 import java.util.List;
 import java.util.Map;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import pro.taskana.classification.internal.jobs.ClassificationChangedJob;
```

Comment:
This dependency is only required because of some constants. Do we really want that? In this case I would get reid of the dependency and declare the constants again. What do you think?

Reply:
Done.

Now we only have dependencies from common.internal.jobs -> [task|workbasket|classification].internal.jobs 

This is necessary, since the AbstractTaskanaJob is a JobFactory